src/Sticky/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$sticky-offset:                       0 !default;
+$sticky-shadow-top:                   0 -0.5rem 1rem rgba(0, 0, 0, 0.15), 0 -0.25rem 0.625rem rgba(0, 0, 0, 0.15) !default;
+$sticky-shadow-bottom:                0 0.5rem 1rem rgba(0, 0, 0, 0.15), 0 0.25rem 0.625rem rgba(0, 0, 0, 0.15) !default;
```

Comment:
Should these be using preexisting elevation SCSS variables, e.g. `$level-1-box-shadow` or `$box-shadow-sm`?

Reply:
I have tried those two variables but they are almost invisible. As it mentioned in the task I needed to copy shadows from the `Scrollable`. But there are custom shadows and they are not put into variables. Then I went to the `core/scss` and didn't find shadows for the `top`. There are only shadows by leveling (they are for the bottom of the block) and no for the positioning (left, top, right). So I went to the figma foundations and found there [Shadows level 3 for top and bottom](https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-Design-System?node-id=9076%3A3783). Transferred them to variables. I think maybe we should move them to the `core/scss`?